mcda/mcda_run.py

```diff
@@ -1,377 +1,165 @@
 #! /usr/bin/env python3
 
-import logging
+"""
+This script serves as the main entry point for running all pieces of functionality in a consequential way by
+following the settings given in the configuration file 'configuration.json'.
+
+Usage (from root directory):
+    $ python3 -m mcda.mcda_run -c configuration.json
+"""
+
 import time
-import sys
-import os
+import logging
 
 from mcda.configuration.config import Config
-from mcda.utils import *
-from mcda.utils_for_parallelization import *
-from mcda.mcda_without_robustness import MCDAWithoutRobustness
-from mcda.mcda_with_robustness import MCDAWithRobustness
+from mcda.utils.utils_for_main import *
+from mcda.utils.utils_for_plotting import *
+from mcda.utils.utils_for_parallelization import *
+
+log = logging.getLogger(__name__)
 
-formatter = '%(levelname)s: %(asctime)s - %(name)s - %(message)s'
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=formatter)
+FORMATTER: str = '%(levelname)s: %(asctime)s - %(name)s - %(message)s'
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=FORMATTER)
 logger = logging.getLogger("ProMCDA")
 
-class UserStoppedInfo(Exception):
-    pass
+# noinspection PyTypeChecker
+def main(input_config: dict):
+    """
+        Execute the ProMCDA (Probabilistic Multi-Criteria Decision Analysis) process.
+
+        Parameters:
+        - input_config (dictionary): Configuration parameters for the ProMCDA process.
```

Comment:
May be include word `dictionary` in the description of the parameter instead of specifying it explicitly in `()`?

Reply:
Solved for now at [c605747](https://github.com/wetransform-os/ProMCDA/commit/c605747e621fa9dc2a98250ca035278e4483eb20). However, next the configuration will be removed.